### ISSUE TITLE
Node-pressure Eviction memory.available script

### ISSUE THREAD

**Issue by BoboceaOctavian** [kubernetes/website#39699]
This is related to https://kubernetes.io/docs/concepts/scheduling-eviction/node-pressure-eviction/#eviction-signals, more specifically to this section:
![image](https://user-images.githubusercontent.com/23451623/221576139-dceebdf0-7089-47ca-b736-7ae1ab87268d.png)

The script file linked here for reproducing the set of steps that the kubelet performs to calculate memory.available (https://github.com/kubernetes/website/blob/main/content/en/examples/admin/resource/memory-available.sh) no longer seems to work with Kubernetes 1.25 and the move to cgroup v2. Lines 10 and 11 generate errors as the memory resources used in them are no longer available.

```
memory_usage_in_bytes=$(cat /sys/fs/cgroup/memory/memory.usage_in_bytes)
memory_total_inactive_file=$(cat /sys/fs/cgroup/memory/memory.stat | grep total_inactive_file | awk '{print $2}')
```

Some output on a v1.25.4 Azure Kubernetes pod created from mcr.microsoft.com/dotnet/aspnet:6.0.13-alpine3.17:
![image](https://user-images.githubusercontent.com/23451623/221583775-3f51b6c0-ae1c-4772-878e-9990802a8b0a.png)

It would be much appreciated to have an update to this script for the new way of calculating memory.

**Comment by sftim** [comment]
I suggest we stop linking to https://github.com/kubernetes/website/blob/main/content/en/examples/admin/resource/memory-available.sh and mark it as deprecated (via a comment in that script).

If someone'd like to recommend and maintain a new way to find out these values, great. However, my gut feeling is that nobody will step up. If that's how it plays out, eventually we can remove the deprecated script.

**Comment by aritraghosh** [comment]
It will be good to have a replacement for the script with cgroup v2

**Comment by aritraghosh** [comment]
I think I have a script which works for  cgroup v2.  I can merge a different one for it

**Comment by sftim** [comment]
/sig node

Should we make this change?
/remove-lifecycle rotten
/lifecycle stale

**Comment by vaibhav2107** [comment]
I think this issue is resolved by #41931 and hence can be closed now
